.github/workflows/modeling.yaml

```diff
@@ -54,38 +55,30 @@ jobs:
     - name: Setup Conda Environment
       uses: mamba-org/setup-micromamba@v1
       with:
-        # default - will pull down 2.0 which we don't want!
-        # micromamba-version: latest
-        # pin to latest 1.x release
-        micromamba-version: '1.5.10-0'
-        environment-file: devtools/conda-envs/asapdiscovery-${{ matrix.os }}.yml
-        environment-name: asapdiscovery
+        micromamba-version: latest
+        environment-file: devtools/conda-envs/${{ matrix.os }}/modeling.yaml
+        environment-name: modeling
         cache-environment: true
         cache-downloads: true
         cache-environment-key: environment-${{ steps.date.outputs.date }}
         cache-downloads-key: downloads-${{ steps.date.outputs.date }}
         create-args: >-
           python==${{ matrix.python-version }}
 
+    - name: Install packages
+      run: |
+        micromamba install -n modeling --file devtools/conda-envs/${{ matrix.os }}/data.yaml
```

Comment:
yes this should work fine! We can come up with a more clever way to do this but for now this is good

Reply:
great, thank you!

Reply:
Yes, this works. This is great. Thanks!